test/unittests/libevm/VMTest.cpp

```diff
@@ -570,6 +570,115 @@ class LegacyVMChainIDTestFixture : public ChainIDTestFixture
     LegacyVMChainIDTestFixture() : ChainIDTestFixture{new LegacyVM} {}
 };
 
+class PrecompileCallFixture : public TestOutputHelperFixture
+{
+public:
+    explicit PrecompileCallFixture(VMFace* _vm) : vm{_vm} { state.addBalance(address, 1 * ether); }
+
+    void testCallHasCorrectCost()
+    {
+        // let r := call(10000, 0x4, 0, 0, 0, 0, 0)
```

Comment:
(Nit) can we document these args e.g.:

call(10000 /* gas  */, 0x4 /* address */, 0 /* value */, ...)

Makes the test code a little more readable 😀